tardis/simulation/base.py

```diff
@@ -383,6 +386,54 @@ def run(self):
         )
         self._call_back()
 
+        self.interactive_widget()
+
+    def interactive_widget(self):
```

Comment:
"interactive widget" isn't very descriptive.  What about "iteration_widget"?

Reply:
Yes, this seems to be more descriptive of what this widget is doing. :) 
I would move it somewhere in the visualization module so that all visualization stuff is in one place 😉